src/components/CurationPage/CurationPage.tsx

```diff
@@ -59,15 +59,21 @@ export default class CurationPage extends React.PureComponent<Props, State> {
   }
 
   renderAssigneeFilterDropdown = () => {
-    const { committeeMembers } = this.props
+    const { committeeMembers, wallet } = this.props
     const { assigneeFilter } = this.state
     return (
       <Dropdown
         direction="left"
         value={assigneeFilter}
         options={[
           { value: ALL_ASSIGNEES_KEY, text: t('curation_page.filter.all_assignees') },
-          ...committeeMembers.map(address => ({ value: address, text: <Profile textOnly address={address} /> }))
+          ...[
```

Comment:
I don't think the first ... is necessary

Reply:
I believe it's still needed because the result of the `reduce` is an `[]` and it needs to be merged with `{ value: ALL_ASSIGNEES_KEY, text: t('curation_page.filter.all_assignees') }`. Right?